lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -72,6 +72,12 @@ def __init__(self, runbook: schema.Platform) -> None:
         self.host_node: Node
         self.vm_disks_dir: str
 
+        # used for port forwarding in case of Remote Host
+        self._next_available_port: int
+        self._port_forwarding_lock: Lock
+        # map env with list of (port, IP) used in port forwading
+        self._env_ports_mapping: Dict[int, List[Tuple[int, str]]]
```

Comment:
If you want to associated data with an Environment object, then store the data within `EnvironmentContext` (`context.py`).

For examples, search for `get_environment_context` within this file.

Reply:
Moved the list to EnvironmentContext.